include/envoy/router/router.h

```diff
@@ -65,6 +65,12 @@ class DirectResponseEntry : public ResponseEntry {
    */
   virtual Http::Code responseCode() const PURE;
 
+  /**
+   * Returns the response flag to record.
+   * @return absl::optional<StreamInfo::ResponseFlag> the response flag.
+   */
+  virtual const absl::optional<StreamInfo::ResponseFlag> responseFlag() const PURE;
```

Comment:
When referring to this field, both here in the function name, in the docs, and everywhere else, please be clear that this is an access log response flag. Otherwise it appears that this is part of the response sent to the client.